src/models.ts

```diff
@@ -198,7 +198,7 @@ export abstract class Filter {
     this.target = target;
   }
 
-  toJSON(): IFilter {
+  toJSON(): any {
```

Comment:
It looks like you changed IFilter but didn't add a replacement for the original interface which is what should be used here.